debian/kafka/include/etc/confluent/docker/run

```diff
@@ -30,7 +26,7 @@ if [ $# -ne 0 ]; then
 fi
 
 echo "===> ENV Variables ..."
-env | sort
+env | sort | grep -vP 'PASSWORD|JAAS_CONFIG'
```

Comment:
what if this list becomes long? May be use a constant string in bash-config so that all such env to be ignored can be listed at one place.

Reply:
this fits in with what I was hoping to see. 

regarding mohnishbasha , its possible this list could grow . 

I also wonder about externalizing this to a common script like the bash_config that was done so customers/support would have an easy way to add env variables that are of particular concern to them.